README.md

```diff
@@ -80,14 +79,23 @@ To try running the examples below, check out the Databricks notebook [DeepLearni
 
 ### Working with images in Spark
 
-The first step to applying deep learning on images is the ability to load the images. Deep Learning Pipelines includes utility functions that can load millions of images into a Spark DataFrame and decode them automatically in a distributed fashion, allowing manipulation at scale.
+The first step to applying deep learning on images is the ability to load the images. Spark and Deep Learning Pipelines include utility functions that can load millions of images into a Spark DataFrame and decode them automatically in a distributed fashion, allowing manipulation at scale.
+
+Using Spark's ImageSchema
 
 ```python
-from sparkdl import readImages
-image_df = readImages("/data/myimages")
+from pyspark.ml.image import ImageSchema
+image_df = ImageSchema.readImages("/data/myimages")
 ```
 
-The resulting DataFrame contains a string column named "filePath" containing the path to each image file, and a image struct ("`SpImage`") column named "image" containing the decoded image data.
+or alternatively, using PIL in python
```

Comment:
yup it does, it produces the same image schema with the same channel ordering. This is the one used in tests.

Reply:
As a user, when should I use the Spark version vs the PIL version? It seems confusing to have both here.

Reply:
OK, I can remove it from the readme. I put it there because we actually did need to use the python read to match Keras so I thought user should know about the option.